test/client/data/dom-utils/iframe.html

```diff
@@ -14,10 +14,10 @@
 </head>
 <body>
 <script type="text/javascript">
-    var HH_SETTINGS = Hammerhead.get('./settings');
-    HH_SETTINGS.set({
-        CROSS_DOMAIN_PROXY_PORT: 2000
-    });
+    var hammerhead = window['%hammerhead%'];
+    var settings = hammerhead.get('./settings');
+
+    settings.get().crossDomainProxyPort = '5555';
```

Comment:
is there any reason to change the port from 2000 to 5555. And why it's a string but not an integer?